src/Private/Pop-WindowsNotification.ps1

```diff
@@ -8,28 +8,43 @@ Function Pop-Notification {
 
         [Parameter(Mandatory=$true)]
         [string]
-        $Title
+        $Title,
+
+        [Parameter()]
+        [string]
+        $Icon = "$PSScriptRoot\lib\alarm.png",
+
+        [Parameter()]
+        [switch]
+        $Silent
```

Comment:
Is there plumbing for `-Silent` yet in `Send-OSNotification`?

Reply:
There's not... yet.

Reply:
I'm inclined to say that we should get rid of that logic until the plumping for `-Silent` is added to all platforms... but since we're not a very complex module yet, I'm fine leaving it in.

Reply:
I opened this: https://github.com/Windos/PoshNotify/issues/9